src/components/App/index.tsx

```diff
@@ -67,61 +68,65 @@ export class AppBase extends React.Component<Props> {
     }
   }
 
-  render() {
+  renderRow(content: JSX.Element, { className = '' } = {}) {
+    return (
+      <Row className={makeClassName(styles.content, className)}>{content}</Row>
+    );
+  }
+
+  renderContent() {
     const { loading, profile } = this.props;
 
-    const routesOrLoginMessage = profile ? (
-      <Switch>
-        <Route exact path="/" component={Index} />
-        <Route
-          component={Browse}
-          exact
-          path="/:lang/browse/:addonId/versions/:versionId/"
-        />
-        <Route component={NotFound} />
-      </Switch>
-    ) : (
-      <p className={styles.loginMessage}>
-        {gettext('Please log in to continue.')}
-      </p>
-    );
+    if (loading) {
+      return this.renderRow(
+        <React.Fragment>
+          <p>{gettext('Getting your workspace ready')}</p>
```

Comment:
Could we add `<p><FontAwesomeIcon icon="spinner" size="3x" spin /></p>` above the text to have the final Windows™ Easter-egg? 🙏🤗

Reply:
whoa, it is totally the Windows 10 spinner, too